### DISCUSSION TITLE
Shopware 6.4.18.1 Compatibility

### DISCUSSION THREAD

**Discussion by nikkoch** [FriendsOfShopware/FroshPlatformThumbnailProcessor#78] [Q&A] [answered]
Shopware 6.4.18.1 Compatibility

Is ThumbnailProcessor incl. Webp-Support compatible with latest Shopware 6.4.18.1? We know, these updates related to security, but would like to be sure.

**Comment by tinect** [comment, marked as answer]
Yes :-)
